core/arch/arm/include/mm/core_memprot.h

```diff
@@ -111,4 +111,9 @@ paddr_t virt_to_phys(void *va);
  */
 vaddr_t get_base(paddr_t base, enum teecore_memtypes type);
 
+/*
+ * Return runtime usable io address
```

Comment:
"Return runtime usable io address, irrespective of whether the MMU is enabled or not. @base has to be in a memory area of type MEM_AREA_IO_SEC or MEM_AREA_IO_NSEC. Returns NULL on failure."